src/xbot/common/subsystems/BasePoseSubsystem.java

```diff
@@ -47,8 +47,7 @@
     
     private BooleanProperty rioRotated;
     
-    @Inject
-    public PoseSubsystem(WPIFactory factory, XPropertyManager propManager) {
+    public BasePoseSubsystem(WPIFactory factory, XPropertyManager propManager) {
```

Comment:
I'm still hesitant to use `TelemetrySource` like this, for two reasons:

- Implementing this interface doesn't actually cause that method to be called; the season-specific code must manually register it.
- The telemetry infrastructure is intended to be for things that are for human use when debugging, but aren't critical to the function of the system; or at least, that is what I had expected it to be. By putting the pose update logic here, we're putting a lot of important functionality in the hands of that debugging-related system.